manual/query_builder/truncate/README.md

```diff
@@ -0,0 +1,16 @@
+## INSERT
+
+To create a TRUNCATE query, use one of the `truncate` methods in [QueryBuilder]. There are
+several variants depending on whether your table name is qualified, and whether you use
+case-sensitive identifiers or case-insensitive strings:
+
+```java
+import static com.datastax.oss.driver.api.querybuilder.QueryBuilder.*;
+
+InsertInto insertCaseInsensitve = truncate("ks", "table");
+InsertInto insertCaseSensitive = truncate(CqlIdentifier.fromCql("ks"), CqlIdentifier.fromCql("table"));
```

Comment:
InsertInto=>Truncate
Also the variables need to be renamed truncateXxx (and notice that there's an "i" missing in the first one CaseInsensit**i**ve).

Could you also add a comment with the generated CQL under each one? See the pages for the other statements for examples of how we've done it so far.